curl.sh

```diff
@@ -1,10 +1,36 @@
 package: curl
-version: "1.0"
-system_requirement_missing: |
-  curl and its development package are missing from your system.
-   * RHEL-compatible systems: you will probably need "curl" and "curl-devel" packages.
-   * Ubuntu-compatible systems: you will probably need "curl" and "libcurl4-openssl-dev" (or "libcurl4-gnutls-dev").
-system_requirement: ".*"
-system_requirement_check: |
-  curl --version > /dev/null; if test $? = 127; then exit 1; else printf "#include <curl/curl.h>\nint main() {}\n" | cc -xc -lcurl - -o /dev/null || exit 2; fi; exit 0
+version: "7.70.0"
+tag: curl-7_70_0
+source: https://github.com/curl/curl.git
+build_requires:
+ - "OpenSSL:(?!osx)"
+ - CMake
+ - alibuild-recipe-tools
 ---
+#!/bin/bash -e
+
+if [[ $ARCHITECTURE = osx* ]]; then
+  OPENSSL_ROOT=$(brew --prefix openssl)
+else
+  ${OPENSSL_ROOT:+env LDFLAGS=-Wl,-R$OPENSSL_ROOT/lib}
+fi
+
+rsync -av --delete --exclude="**/.git" $SOURCEDIR/ .
+
+./buildconf
+./configure --prefix=$INSTALLROOT ${OPENSSL_ROOT:+--with-ssl=$OPENSSL_ROOT} --disable-static
+make ${JOBS:+-j$JOBS}
+make install
+
+# Modulefile
+mkdir -p etc/modulefiles
+alibuild-generate-module > etc/modulefiles/$PKGNAME
+cat > etc/modulefiles/$PKGNAME <<EoF
```

Comment:
This will delete the modulefile that you generate above. I guess it's not intended?

Reply:
Thanks, my intention is to append new lines to modulefile.